server/re/src/mailMS.cpp

```diff
@@ -70,8 +61,8 @@ int msi_enable_msiSendMail(MsParam* _value, RuleExecInfo* _rei)
  * \note This microservice sends e-mail using the mail command in the unix system. No attachments are supported. The
  *       sender of the e-mail is the unix user-id running the irodsServer.
  *
- * \warning This microservice must be enabled via msi_enable_msiSendMail first, otherwise the call to msiSendMail will
- * fail.
+ * \warning All invocations of this microservice must be preceded by a call to msi_enable_msiSendMail, otherwise the
+ * call to msiSendMail will fail.
```

Comment:
will it fail?  or just exit silently and 'early'?

Reply:
oh, i see, it's an actual error now.

Reply:
Yep. If the MSI to enable use isn't called first, `msiSendMail` will report an error.